src/Files.App/Views/LayoutModes/BaseLayout.cs

```diff
@@ -483,16 +470,17 @@ public void SetSelectedItemsOnNavigation()
 					navigationArguments.SelectItems is not null &&
 					navigationArguments.SelectItems.Any())
 				{
-					List<ListedItem> liItemsToSelect = new();
+					List<ListedItem> listedtemsToSelect = new();
 					foreach (string item in navigationArguments.SelectItems)
-						liItemsToSelect.Add(ParentShellPageInstance!.FilesystemViewModel.FilesAndFolders.Where((li) => li.ItemNameRaw == item).First());
+						listedtemsToSelect.Add(ParentShellPageInstance!.FilesystemViewModel.FilesAndFolders.Where((li) => li.ItemNameRaw == item).First());
```

Comment:
It can be written on one line using `AddRange` instead of `Add`. `Contains` can be used for the `Where` condition.

Reply:
@hishitetsu is this resolved?

Reply:
No. This is a suggestion for performance improvement.

Reply:
Can you add a code suggestion and I'll accept?